taxi/views.py

```diff
@@ -85,6 +134,30 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
     model = Driver
     paginate_by = 5
 
+    def get_context_data(
+            self,
+            *,
+            object_list: list = None,
```

Comment:
add additional typehint for list inner objects like list[ANY_TYPE]
use typehint like: <ANY_TYPE> | None = None